mypy/types.py

```diff
@@ -341,8 +342,8 @@ def accept(self, visitor: 'TypeVisitor[T]') -> T:
         return visitor.visit_any(self)
 
     def copy_modified(self,
-                      type_of_any: TypeOfAny = _dummy,
-                      original_any: Optional['AnyType'] = _dummy,
+                      type_of_any: Bogus[TypeOfAny] = _dummy,
+                      original_any: Bogus[Optional['AnyType']] = _dummy,
```

Comment:
Maybe add also a _short_ comment about `Bogus` here (just recall that `_dummy` is `Any`)?